.github/workflows/coverall.yml

```diff
@@ -20,6 +20,6 @@ jobs:
         npm install
         npm run test
     - name: Coveralls
-      uses: coverallsapp/github-action@v1.1.2
+      uses: coverallsapp/github-action@master
```

Comment:
If you use `v1` then you will have latest version from `v1`.

Reply:
I know, but github runners are using a cached version of `@v1.1.2` which contains an underlying dependency cache that has not been updated (they broke a version). So if I use `1.1.2` at times the run will fail.

related issues https://github.com/nickmerwin/node-coveralls/issues/280 and https://github.com/coverallsapp/github-action/issues/76

Reply:
Which was why https://github.com/asyncapi/generator-model-sdk/runs/2618803562 failed

Reply:
Ok